packages/teams-js/src/public/interfaces.ts

```diff
@@ -179,76 +179,97 @@ export enum FileOpenPreference {
  *
  * Represents the structure of the received context message.
  */
+
 export interface Context {
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.groupId} instead
```

Comment:
These mappings aren't correct. The `Context.groupId` parameter actually maps to `app.Context.team.groupId`. 

You can check the URL that is referenced on line 178 for all the mappings or you can take a look at transformLegacyContextToAppContext function in app.ts for how they map.

Reply:
my mistake, I was using the auto completion, so that is my fault.  will definitely change, and you had said to check that function

Reply:
Cool thanks. Yeah, the example I sent in chat did point to `app.Context.team.groupId`. :)